core.serge.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "name": "meter",
+    "source_dir": "components/meter/lang",
+    "output_file_path": "components/meter/%LANG%.json",
+    "parser_plugin": {
+      "plugin": "parse_d2l_fra"
+    },
+    "output_lang_rewrite": [
+      "ar-sa ar",
+      "da-dk da-dk",
+      "de-de de",
+      "en en",
+      "es-mx es",
+      "fi-FI fi",
+      "fr-ca fr",
+      "ja-jp ja",
+      "ko-kr ko",
+      "nl-nl nl",
+      "pt-br pt",
+      "sv-se sv",
+      "tr-tr tr",
+      "zh-cn zh",
+      "zh-tw zh-tw"
+    ],
+    "pr_reviewers": [
+      "m6jones"
+    ]
+  }
+]
```

Comment:
What happens to this file when there are more translations beyond meter?

Reply:
You add another object (similar to the meter one) to the list.